Header header
bool OrientationAsQaut
Pose[] PoseArray
time[] t_k #desired execution of the trajectory
